Dockerfile

```diff
@@ -180,6 +180,82 @@ RUN npm run production
 RUN rm -rf node_modules
 
 
+###
+
+# ⛓️‍💥 NPM supply chain attack note: Trust Docker Hub and Deno to provide a safe container image.
+FROM denoland/deno AS deno-base
+
+WORKDIR /app
+
+# Set the path for the esbuild binary, accessed in the Dockerfile and build.js
+ENV ESBUILD_BINARY_PATH=/usr/local/bin/esbuild
+
+# Install curl and ca-certificates for downloading esbuild
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    curl \
+    ca-certificates
+
+# Install, verify and extract esbuild
+# The version should match the one used in build.js
+
+# ⛓️‍💥 NPM supply chain attack note: Trust esbuild to have published a safe package (esbuild binary) to npm
+RUN curl https://registry.npmjs.org/@esbuild/linux-x64/-/linux-x64-0.25.10.tgz -o /tmp/esbuild-linux-x64.tgz
```

Comment:
@earthlingdavey are you gonna use this separately or is this to "prefetch" esbuild for Deno itself (using `deno bundle`)?